js/app.js

```diff
@@ -13,6 +13,8 @@ var Tracker = function () {
 	this.photoArray = [];
 }
 
+var tracker = new Tracker();
```

Comment:
Yep, I just tried this and I still have a 4 space tab. And as soon as I save the Preferences, I get the following error:
Error trying to parse settings: Unexpected trailing characters in ~/Library/Application Support/Sublime Text 2/Packages/User/Preferences.sublime-settings:10:1
But I can't find any trailing characters.
I'll keep working on it though! I'll just use the Googles!